features/plant/managmentPlantAsGuest.feature

```diff
@@ -0,0 +1,33 @@
+Feature: Manage plant as guest
+  In order to manage plant
+  As a anonymous
+  I need to be able to log in
+
+  Scenario: Adding plant as guest
+    When I am on "plant/new"
+    Then I should see "Login"
+    And the response status code should not be 404
+    And the response status code should not be 403
+  Scenario: Show my plants as guest
+    When I am on "plant/all"
+    Then I should see "Login"
+    And the response status code should not be 404
```

Comment:
It SHOULD BE 200. This kind of tests are good when you expect one of many codes, but one or two. 

Here you just expect 200 OK.